product-service/postProduct.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { Client } = require('pg');
+const schema = require('./model/Product');
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbParams = {
+  host: PG_HOST,
+  port: PG_PORT,
+  database: PG_DATABASE,
+  user: PG_USERNAME,
+  password: PG_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false
+  },
+  connectionTimeoutMillis: 5000
+}
+
+const errorResp = (errors) => {
+    const err = errors.map(err => {
+      const { path, message } = err
+      return { path, message }
+    })
+    return {
+      status: 'validation is failed',
+      err
+    }
+  }
+
+module.exports.postProduct = async event => {
+
+  console.log(`Request: ${JSON.stringify(event)}`);
+  const body = JSON.parse(event.body);
+  const { error } = schema.validate(body);
+  if (error) {
+      return {
```

Comment:
Validation seams to work a bit strange (in this case it is 400 status?)
<img width="918" alt="Screenshot 2020-11-09 at 12 43 56" src="https://user-images.githubusercontent.com/10729933/98525405-77296e80-2289-11eb-8330-1e18e38b4d92.png">

Reply:
Thank you. Fixed